lottery/mod.go

```diff
@@ -0,0 +1,23 @@
+package lottery
+
+import (
+	"go.dedis.ch/dela/ledger/arc"
+	"go.dedis.ch/kyber/v3"
+)
+
+// Secret defines the primitives to run a Calypso-like app. In the case of
+// Calypso it is mainly a wrapper arround DKG that provides a storage and
+// authorization layer.
+type Secret interface {
```

Comment:
I'm not sure to understand why those two interfaces are part of the lottery package as it is not specific to a lottery. You can simply have a calypso package with a _Calypso_ interface and an _EncryptedMessage_ interface. Let's keep applications out of the repo (or maybe in a integration test).

Reply:
You can refer to the diagram and update it if necessary.

Reply:
Yep, lottery is not the best name. I did this to avoid having a `calypso.Calypso` interface, which isn't recommended.

Reply:
To be fair, `calypso.Calypso` does not get any complain from go lint and it's much clearer than `lottery.Secret`.